os/hal/src/hal_usb_msd.c

```diff
@@ -393,6 +407,12 @@ void msdStart(USBMassStorageDriver *msdp, USBDriver *usbp,
   else {
     msdp->scsi_config.inquiry_response = inquiry;
   }
+  if (NULL == serialInquiry) {
```

Comment:
Reverse?

Reply:
Why reverse? It is harmless. I often use this order to defend me from myself. I know about warnings from smart modern compilers but my habit came from ancient time.